swap_meet/vendor.py

```diff
@@ -1,2 +1,112 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if not inventory: # or if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    
+    def get_by_category(self, category1):
```

Comment:
`category1` I think could be left as `category` or made into a more descriptive name if you want to change it.